lib/eth/version.rb

```diff
@@ -22,7 +22,7 @@ module Eth
   MINOR = 5.freeze
 
   # Defines the patch version of the {Eth} module.
-  PATCH = 13.freeze
+  PATCH = 14.freeze
```

Comment:
Can you revert this? I already bumped to 13 🙏🏼 

```suggestion
  PATCH = 13.freeze
```

Reply:
Got it